src/Client.php

```diff
@@ -625,8 +625,8 @@ public function deleteRoute(string $name) {
   /**
    * {@inheritdoc}
    */
-  public function getBuilds(string $name) {
-    return $this->apiCall(__METHOD__, '', 'buildconfig=' . $name);
+  public function getBuilds(string $name, string $label) {
+    return $this->apiCall(__METHOD__, '', $name);
```

Comment:
Err what ? Use the name as the label ?? Can we have an explanation for what this is doing

Reply:
Guessing it's supposed to be $label